kahuna/public/js/upload/dnd-uploader.js

```diff
@@ -170,9 +179,22 @@ dndUploader.directive('dndUploader', ['$window', 'delay', 'safeApply', 'track',
                         ctrl.importing = false;
                     });
                     track.action(trackEvent, dropAction('Witness'));
-                } else {
+                } else if (uri) {
+                    ctrl.importing = true;
+                    ctrl.loadUriImage(uri)
+                        .catch(error => {
+                            if (error.body && error.body.errorMessage) {
+                                $window.alert(error.body.errorMessage);
```

Comment:
What is that in the case of dragging either something that is not an image, or an image that isn't a JPG? Let's just make sure the message is understandable to human beings (aka non programmers), like @akash1810 did for the upload error.